packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+      <span>Length</span>
+      <span>4.6-39.6 IN</span>
+      <span>11.7-100.6 cm</span>
+    </div>
+    <div>
+      <span>Diet</span>
+      <span>Omnivore</span>
+      <span>nuts, fruits, eggs</span>
+    </div>
+  </CardBody>
+  <CardFooter>
+    <Button>Share</Button>
+  </CardFooter>
+</Card>
+```
+
+## Behaviours
+
+### Mouse
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the mouse](./assets/selectable-interaction-mouse.png)
+
+Clicking the card surface can toggle the checkbox state to `selected`. Note that checkboxes are composed together as an additional component and are not provided by default.
+
+1. Rest Default state is `unselected`
+2. Hover cursor style changes to `pointer`, background color and elevation follows interactive styling to show visual feedback
+3. Pressed cursor does not change background color and elevation follows interactive styling to show visual feedback for pressed state.
+4. MouseUp Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the mouse](./assets/non-interactive-interaction-mouse.png)
+
+Cards that have multiple actionable child components within them cannot be clickable. We preserve the use of `EnterKey` for moving focus to actionable controls inside the card, therefore the card surface cannot be used to set the state of the card but is reliant on the checkbox or similar inputs to set the card state.
+
+Mouse interaction
+
+- OnMouseEnter / Hover cursor style stays at default, rest state stays persistent no visual feedback is applied to the card surface
+- Hover & onClick (Checkbox) Hover state & pressed state applies to checkbox component
+- MouseUp (Checkbox) Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Context menu
+
+![Visual interaction of the context menu on cards using the mouse](./assets/context-interaction-mouse.png)
+
+A context menu can be triggered by right clicking the surface or through an action in the card.
+
+1.Rest
+
+2. RightClick [LongPress]
+   Right click anywhere on card surface invokes content menu
+3. Show
+   Context menu appears close to cursor location on card
+4. Hide/Dismiss
+   Clicking or tapping anywhere outside of menu to dismiss
+
+#### Disabled
+
+![Visual interaction of disabled cards using the mouse](./assets/disabled.png)
+
+1 Unselected disabled
+
+1. Label: Selectable Card
+2. Role: Group
+3. State: Disabled, unselected
+
+2 Selected disabled
+
+1. Label: Selectable Card
+2. Role: Group
+3. State: Disabled, selected
+
+### Keyboard
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the keyboard](./assets/selectable-interaction-keyboard.png)
+
+1. Rest Default state is ‘unselected’
+2. Focus Focus rectangle is shown on card
+3. Focus Use spacebar to toggle state (selected/unselected)
+
+### Non-interactive cards
```

Comment:
The heading says non-interactive, but below you describe interaction with focus inside. Is it wrong heading?

Reply:
I think it means that the Cards themselves are not interactive but the contents can be. Maybe non-clickable Cards would be a better heading?

Reply:
What @khmakoto said. Non-interactive cards are cards that do not have an action when being clicked on (no onClick defined), think for example a Microsoft Store scenario where you have apps in cards and clicking on them would open a new page.
What do you mean by focus inside?